src/blocks/testimonials/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import { TestimonialIcon as icon } from '@godaddy-wordpress/coblocks-icons';
+
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon } from '@wordpress/components';
+
+/**
+ * Internal dependencies.
+ */
+import edit from './edit';
+import metadata from './block.json';
+import save from './save';
+
+/**
+ * Block constants.
+ */
+const { name, category, attributes } = metadata;
+
+const settings = {
+	attributes,
+	/* translators: block description */
+	description: __( 'Display a list of testimonials.', 'coblocks' ),
+	edit,
+	example: {
+		innerBlocks: [
+			{
+				attributes: {
+					description: 'Lorem ipsum dolor sit amet, consec tetur adipiscing elit. Nam condimentum tempus diam, ultricies sollicitudin erat facilisis eget. Vestibulum rhoncus dui vel eros laoreet consectetur.',
+					name: 'John Doe',
+					title: 'Senior Director',
+				},
+				name: 'coblocks/testimonial',
+			},
+			{
+				attributes: {
+					description: 'Lorem ipsum dolor sit amet, consec tetur adipiscing elit. Nam condimentum tempus diam, ultricies sollicitudin erat facilisis eget. Vestibulum rhoncus dui vel eros laoreet consectetur.',
+					name: 'John Doe',
+					title: 'Senior Director',
+				},
+				name: 'coblocks/testimonial',
+			},
+		],
+	},
+	getEditWrapperProps( atts ) {
+		return { 'data-columns': atts.columns };
+	},
+	icon: <Icon icon={ icon } />,
+	keywords: [
+		'coblocks',
+		/* translators: block keyword */
+		__( 'reviews', 'coblocks' ),
+		/* translators: block keyword */
+		__( 'testimonials', 'coblocks' ),
+	],
+	save,
+	supports: {
+		align: [ 'wide' ],
+	},
```

Comment:
These block supports are now defined in block.json. We should remove this declaration here and instead place it in the block.json. Here is an example where the block supports for `align` defined in the block.json. I might advise exploring allowing `none` `wide` and `full` alignments for more aesthetic flexibility. 

https://github.com/WordPress/gutenberg/blob/004a32ec09d5b789be694d85725fe49cca385f08/packages/block-library/src/group/block.json#L10-L14

https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/#align

We are not actually doing this correctly in any block as it turns out even the newest block `Counter` that I made. That block I mistakenly defined an `align` attribute and did not opt into the align supports. This is a failure because there is a good chance that this block will eventually need to be deprecated so that we can properly opt-in to the `align` block support. It looks like you don't even need to import any custom components or anything just opt-in 🤦‍♂️